app/controllers/api/v1/peers/search_controller.rb

```diff
@@ -17,23 +17,29 @@ def index
   private
 
   def require_enabled_api!
-    head 404 unless Setting.peers_api_enabled && Chewy.enabled? && !whitelist_mode?
+    head 404 unless Setting.peers_api_enabled && !whitelist_mode?
```

Comment:
Based on this, confirming as I don't see it explicitly mentioned:

If one unchecks `Publish list of discovered servers in the API` (disabling the peers API), the auto-completion for this interaction model is also disabled, avoiding leaking known instances?

Reply:
Yes